src/libraries/Microsoft.PowerFx.Interpreter/UDF/DefineFunctionsResult.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Interpreter.UDF
+{
+    public class DefineFunctionsResult
```

Comment:
>DefineFunctionsResult [](http://example.com/codeflow?start=17&length=21)

Can this impl IOperationStatus, like CheckResult.

Reply:
Done